### ISSUE TITLE
10.11: Error Writing NBImageInfo.plist

### ISSUE THREAD

**Issue by macmule** [macmule/AutoCasperNBI#61]
10.11's PlatformSupport.plist, whilst still located at /System/Library/CoreServices/PlatformSupport.plist, only contains the "SupportedBoardIds" array & not also the "SupportedModelProperties" as other OS's.

This error is due to me looking for the "SupportedBoardIds" array.

I'll log a bug with Apple regarding this change & think about a solution.

10.9's PlatformSupport.plist below:
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>SupportedBoardIds</key>
	<array>
		<string>Mac-031B6874CF7F642A</string>
		<string>Mac-F2268DC8</string>
		<string>Mac-F22587C8</string>
		<string>Mac-50619A408DB004DA</string>
		<string>Mac-F2218EA9</string>
		<string>Mac-742912EFDBEE19B3</string>
		<string>Mac-F22C8AC8</string>
		<string>Mac-AFD8A9D944EA4843</string>
		<string>Mac-F227BEC8</string>
		<string>Mac-F226BEC8</string>
		<string>Mac-F305150B0C7DEEEF</string>
		<string>Mac-35C1E88140C3E6CF</string>
		<string>Mac-77EB7D7DAF985301</string>
		<string>Mac-2E6FAB96566FE58C</string>
		<string>Mac-7BA5B2794B2CDB12</string>
		<string>Mac-BE0E8AC46FE800CC</string>
		<string>Mac-00BE6ED71E35EB86</string>
		<string>Mac-F42D86A9</string>
		<string>Mac-4B7AC7E43945597E</string>
		<string>Mac-F22C89C8</string>
		<string>Mac-F22587A1</string>
		<string>Mac-937CB26E2E02BB01</string>
		<string>Mac-942459F5819B171B</string>
		<string>Mac-F42388C8</string>
		<string>Mac-F223BEC8</string>
		<string>Mac-F4238CC8</string>
		<string>Mac-F222BEC8</string>
		<string>Mac-4BC72D62AD45599E</string>
		<string>Mac-F2208EC8</string>
		<string>Mac-66F35F19FE2A0D05</string>
		<string>Mac-F4238BC8</string>
		<string>Mac-189A3D4F975D5FFC</string>
		<string>Mac-C08A6BB70A942AC2</string>
		<string>Mac-8ED6AF5B48C039E1</string>
		<string>Mac-FA842E06C61E91C5</string>
		<string>Mac-FC02E91DDD3FA6A4</string>
		<string>Mac-06F11FD93F0323C5</string>
		<string>Mac-F42C86C8</string>
		<string>Mac-06F11F11946D27C5</string>
		<string>Mac-6F01561E16C75D06</string>
		<string>Mac-F60DEB81FF30ACF6</string>
		<string>Mac-81E3E92DD6088272</string>
		<string>Mac-F2268EC8</string>
		<string>Mac-F22589C8</string>
		<string>Mac-3CBD00234E554E41</string>
		<string>Mac-F22788AA</string>
		<string>Mac-F22586C8</string>
		<string>Mac-F221BEC8</string>
		<string>Mac-942C5DF58193131B</string>
		<string>Mac-F2238BAE</string>
		<string>Mac-9F18E312C5C2BF0B</string>
		<string>Mac-F2268CC8</string>
		<string>Mac-C3EC7CD22292981F</string>
		<string>Mac-F221DCC8</string>
		<string>Mac-F2218FC8</string>
		<string>Mac-94245A3940C91C80</string>
		<string>Mac-27ADBB7B4CEE8E61</string>
		<string>Mac-F65AE981FFA204ED</string>
		<string>Mac-F42D89C8</string>
		<string>Mac-942B59F58194171B</string>
		<string>Mac-F42D89A9</string>
		<string>Mac-F2268AC8</string>
		<string>Mac-F22C86C8</string>
		<string>Mac-F2238AC8</string>
		<string>Mac-942452F5819B1C1B</string>
		<string>Mac-F2218FA9</string>
		<string>Mac-F42D88C8</string>
		<string>Mac-94245B3640C91C81</string>
		<string>Mac-F42D86C8</string>
		<string>Mac-42FD25EABCABB274</string>
		<string>Mac-7DF2A3B5E5D671ED</string>
		<string>Mac-F2268DAE</string>
		<string>Mac-2BD1B31983FE1663</string>
		<string>Mac-7DF21CB3ED6977E5</string>
		<string>Mac-031AEE4D24BFF0B1</string>
		<string>Mac-F42C88C8</string>
		<string>Mac-35C5E08120C7EEAF</string>
		<string>Mac-F42386C8</string>
		<string>Mac-E43C1C25D4880AD6</string>
		<string>Mac-F42C89C8</string>
		<string>Mac-942B5BF58194151B</string>
		<string>Mac-F2218EC8</string>
	</array>
	<key>SupportedModelProperties</key>
	<array>
		<string>MacPro6,1</string>
		<string>MacBookPro4,1</string>
		<string>Macmini5,3</string>
		<string>Macmini5,2</string>
		<string>Macmini5,1</string>
		<string>iMac10,1</string>
		<string>MacPro4,1</string>
		<string>MacBookPro5,2</string>
		<string>iMac8,1</string>
		<string>MacBookPro5,5</string>
		<string>MacBookAir4,2</string>
		<string>iMac11,1</string>
		<string>MacBookPro7,1</string>
		<string>iMac11,3</string>
		<string>MacBookPro8,2</string>
		<string>MacBookPro3,1</string>
		<string>iMac13,2</string>
		<string>iMac13,3</string>
		<string>MacPro5,1</string>
		<string>iMac9,1</string>
		<string>Macmini3,1</string>
		<string>iMac13,1</string>
		<string>Macmini7,1</string>
		<string>iMac12,2</string>
		<string>MacBook7,1</string>
		<string>iMac12,1</string>
		<string>MacBook5,1</string>
		<string>MacBook5,2</string>
		<string>iMac11,2</string>
		<string>MacBookPro5,1</string>
		<string>Macmini6,1</string>
		<string>Macmini6,2</string>
		<string>MacBookPro11,3</string>
		<string>MacBookPro11,2</string>
		<string>MacBookPro11,1</string>
		<string>MacBookPro6,2</string>
		<string>MacBookPro10,2</string>
		<string>MacBookPro10,1</string>
		<string>iMac14,4</string>
		<string>MacBookPro9,2</string>
		<string>iMac14,1</string>
		<string>iMac14,3</string>
		<string>iMac14,2</string>
		<string>MacBookPro6,1</string>
		<string>MacBookPro5,4</string>
		<string>iMac15,1</string>
		<string>MacBookAir3,1</string>
		<string>MacBookAir3,2</string>
		<string>Macmini4,1</string>
		<string>Xserve3,1</string>
		<string>MacBookAir2,1</string>
		<string>MacBookAir6,1</string>
		<string>MacBookAir6,2</string>
		<string>MacBookPro8,1</string>
		<string>MacBookAir4,1</string>
		<string>MacBookPro8,3</string>
		<string>iMac7,1</string>
		<string>MacBookPro9,1</string>
		<string>MacBook6,1</string>
		<string>MacBookPro5,3</string>
		<string>MacBookAir5,2</string>
		<string>MacPro3,1</string>
		<string>MacBookAir5,1</string>
	</array>
</dict>
</plist>
```

10.10 PlatformSupport.plist
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>SupportedBoardIds</key>
	<array>
		<string>Mac-031B6874CF7F642A</string>
		<string>Mac-F2268DC8</string>
		<string>Mac-F22C86C8</string>
		<string>Mac-F22587C8</string>
		<string>Mac-50619A408DB004DA</string>
		<string>Mac-F2218EA9</string>
		<string>Mac-742912EFDBEE19B3</string>
		<string>Mac-F22C8AC8</string>
		<string>Mac-F22586C8</string>
		<string>Mac-AFD8A9D944EA4843</string>
		<string>Mac-F227BEC8</string>
		<string>Mac-F226BEC8</string>
		<string>Mac-F305150B0C7DEEEF</string>
		<string>Mac-35C1E88140C3E6CF</string>
		<string>Mac-77EB7D7DAF985301</string>
		<string>Mac-2E6FAB96566FE58C</string>
		<string>Mac-7BA5B2794B2CDB12</string>
		<string>Mac-BE0E8AC46FE800CC</string>
		<string>Mac-00BE6ED71E35EB86</string>
		<string>Mac-F42D86A9</string>
		<string>Mac-4B7AC7E43945597E</string>
		<string>Mac-F22C89C8</string>
		<string>Mac-F22587A1</string>
		<string>Mac-937CB26E2E02BB01</string>
		<string>Mac-942459F5819B171B</string>
		<string>Mac-F42388C8</string>
		<string>Mac-F223BEC8</string>
		<string>Mac-F4238CC8</string>
		<string>Mac-F222BEC8</string>
		<string>Mac-4BC72D62AD45599E</string>
		<string>Mac-F2208EC8</string>
		<string>Mac-66F35F19FE2A0D05</string>
		<string>Mac-F4238BC8</string>
		<string>Mac-189A3D4F975D5FFC</string>
		<string>Mac-C08A6BB70A942AC2</string>
		<string>Mac-8ED6AF5B48C039E1</string>
		<string>Mac-FA842E06C61E91C5</string>
		<string>Mac-FC02E91DDD3FA6A4</string>
		<string>Mac-06F11FD93F0323C5</string>
		<string>Mac-06F11F11946D27C5</string>
		<string>Mac-6F01561E16C75D06</string>
		<string>Mac-F60DEB81FF30ACF6</string>
		<string>Mac-81E3E92DD6088272</string>
		<string>Mac-F2268EC8</string>
		<string>Mac-F22589C8</string>
		<string>Mac-3CBD00234E554E41</string>
		<string>Mac-F22788AA</string>
		<string>Mac-F42C86C8</string>
		<string>Mac-F221BEC8</string>
		<string>Mac-942C5DF58193131B</string>
		<string>Mac-F2238BAE</string>
		<string>Mac-9F18E312C5C2BF0B</string>
		<string>Mac-F2268CC8</string>
		<string>Mac-C3EC7CD22292981F</string>
		<string>Mac-F221DCC8</string>
		<string>Mac-F2218FC8</string>
		<string>Mac-94245A3940C91C80</string>
		<string>Mac-27ADBB7B4CEE8E61</string>
		<string>Mac-F65AE981FFA204ED</string>
		<string>Mac-F42D89C8</string>
		<string>Mac-942B59F58194171B</string>
		<string>Mac-F42D89A9</string>
		<string>Mac-F2268AC8</string>
		<string>Mac-F42C89C8</string>
		<string>Mac-F2238AC8</string>
		<string>Mac-942452F5819B1C1B</string>
		<string>Mac-F2218FA9</string>
		<string>Mac-F42D88C8</string>
		<string>Mac-94245B3640C91C81</string>
		<string>Mac-F42D86C8</string>
		<string>Mac-42FD25EABCABB274</string>
		<string>Mac-7DF2A3B5E5D671ED</string>
		<string>Mac-F2268DAE</string>
		<string>Mac-2BD1B31983FE1663</string>
		<string>Mac-7DF21CB3ED6977E5</string>
		<string>Mac-031AEE4D24BFF0B1</string>
		<string>Mac-F42C88C8</string>
		<string>Mac-35C5E08120C7EEAF</string>
		<string>Mac-F42386C8</string>
		<string>Mac-E43C1C25D4880AD6</string>
		<string>Mac-942B5BF58194151B</string>
		<string>Mac-F2218EC8</string>
	</array>
</dict>
</plist>
```

**Comment by macmule** [comment]
Logged Bug with Apple, (22677569), & have had it closed as a dupe of 22799473.

**Comment by macmule** [comment]
See: https://macmule.com/2015/10/02/autocaspernbi-autoimagrnbi-el-capitan/#Update-2
